dev/io.openliberty.thirdparty.faces-3.0/bnd.bnd

```diff
@@ -35,3 +35,5 @@ publish.wlp.jar.suffix: dev/api/third-party
 
 -buildpath: io.openliberty.org.apache.myfaces.3.0;version=project, \
     org.apache.myfaces.core:myfaces-impl;version=3.0.0,\
+    io.openliberty.jakarta.faces.3.0;version=latest,\
+    io.openliberty.jakarta.el.4.0,\
```

Comment:
Why are changes necessary in the io.openliberty.thirdparty.faces-3.0/bnd.bnd? In addition you'll want to rebase on integration since I recently renamed el.4.0 to expressionLanguage.4.0

Reply:
Sounds good Ill rebase and make the changes. So when I added the overlay class and built the project I got some compilation errors which were resolved when pulling in those projects at the build path and thats why I added them. Not sure why it didnt hit some errors before

Reply:
Ah ok looking at the third party bnd.bnd:

Export-Package: \
  org.apache.myfaces.renderkit.html;thread-context=true;version=3.0.0, \
  org.apache.myfaces.shared.config;version=3.0.0, \
  org.apache.myfaces.shared.renderkit;version=3.0.0, \
  org.apache.myfaces.shared.renderkit.html;version=3.0.0, \
  org.apache.myfaces.shared.renderkit.html.util;version=3.0.0

So probably since we're now compiling a class in org.apache.myfaces.shared.renderkit.html since you have an overlay is why we needed these updates.

Reply:
Made the changes to include the overlays in the core **io.openliberty.org.apache.myfaces.3.0** bundle and then thirdparty pulls in the changes from the project which will then get the overlay. Tested locally and the updated to jakarta are being made this way